README.md

```diff
@@ -54,6 +54,8 @@ so you shouldn't be serving stale redirects.
 `srm_max_redirects` that you can utilize to up this number.
 * "Redirect From" and requested paths are case insensitive by default.
 * Developers can use `srm_additional_status_codes` filter to add status codes if needed.
+* Rules set with 403 and 410 status codes are handled by applying the HTTP status code and rendering a blank page.
```

Comment:
Seems like this isn't 100% true anymore. There is an option to render a message instead of just a blank page (would also need updating in readme.txt)